dotCMS/src/curl-test/BringBack.postman_collection.json

```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "07fa881b-476f-49a9-9e00-997cc61e4840",
+		"_postman_id": "bcb92a0a-19ee-4710-8120-847372062cac",
```

Comment:
Revert this change, as it is not necessary.

Reply:
This has been addressed